x/mint/keeper/keeper_test.go

```diff
@@ -336,6 +336,12 @@ func (suite *KeeperTestSuite) TestSetInitialSupplyOffsetDuringMigration() {
 			bankKeeper := suite.App.BankKeeper
 			mintKeeper := suite.App.MintKeeper
 
+			// in order to ensure the offset is correctly calculated, we need to mint the supply + 1
+			// this is because a negative supply offset will always return zero
+			// by setting this to the supply + 1, we ensure we are correctly calculating the offset by keeping it delta positive
+			supply := bankKeeper.GetSupply(ctx, sdk.DefaultBondDenom)
+			mintKeeper.MintCoins(ctx, sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, supply.Amount.Add(sdk.NewInt(1)))))
```

Comment:
I think we should instead mint the amount that is equal to `keeper.DeveloperVestingAmount`. While these are equal with supply, it might be less apparent why we are using supply here. 

The ultimate reason for having the offset in the first place is having developer vesting rewards so using the amount related to it might be more obvious in terms of why this is done

Reply:
This is a really good catch, thank you!